string input_xml
---
string output_xml
